src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1246,4 +1228,16 @@ void primitive_inst::load(cldnn::BinaryInputBuffer& ib) {
         _intermediates_memory[i] = get_network().get_engine().allocate_memory(ibuf_layout, _allocation_type);
     }
 }
+
+size_t primitive_inst::get_layout_key(const kernel_impl_params& params) const {
+    size_t seed = _node->hash();
```

Comment:
Now it has more info in the hash than layout. 
Would be better to rename the function :)

Reply:
Change func name from get_layout_key to get_impl_key